server/business/bill.business.js

```diff
@@ -0,0 +1,36 @@
+﻿const DataLayer = require("../datalayer/mongo");
+const model = require("../database").getModel("bill");
+const httpError = require("http-errors");
+
+module.exports = class billBusiness {
+ constructor() {
+  // Create an instance of the data layer.
+  this.dataLayer = new DataLayer(model);
+ }
+
+ /**
+  *  Get all bills.
+  */
+ async getAllBills(queryString) {
+  const filter = {
+   driver: queryString.driver,
+   paid: queryString.paid,
+   limit: queryString.limit ?? 10,
```

Comment:
Could we add a //TODO here so we don't forget to update the number down the line